robocrys/condense/mineral.py

```diff
@@ -0,0 +1,245 @@
+"""
+This module provides tools for matching structures to known mineral class.
+"""
+
+from itertools import islice
+from typing import List, Optional, Dict, Text, Any
+
+import numpy as np
+from matminer.utils.io import load_dataframe_from_json
+from pkg_resources import resource_filename
+
+from pymatgen.analysis.aflow_prototypes import AflowPrototypeMatcher
+from pymatgen.core.structure import IStructure
+from robocrys.condense.fingerprint import (get_structure_fingerprint,
+                                           get_fingerprint_distance)
+
+
+class MineralMatcher(object):
+    """Class to match a structure to a mineral name.
+
+    Uses a precomputed database of minerals and their fingerprints, extracted
+    from the AFLOW prototype database. For more information on this database
+    see reference [aflow]_:
+
+    .. [aflow] Mehl, M. J., Hicks, D., Toher, C., Levy, O., Hanson, R. M., Hart,
+               G., & Curtarolo, S. (2017), The AFLOW library of crystallographic
+               prototypes: part 1. Computational Materials Science, 136,
+               S1-S828. doi: 10.1016/j.commatsci.2017.01.017
+
+    Args:
+        aflow_initial_ltol: The fractional length tolerance used in the AFLOW
+            structure matching.
+        aflow_initial_stol : The site coordinate tolerance used in the AFLOW
+            structure matching.
+        aflow_initial_angle_tol: The angle tolerance used in the AFLOW structure
+            matching.
+        use_fingerprint_matching: Whether to use the fingerprint distance to
+            match minerals.
+        fingerprint_distance_cutoff: Cutoff to determine how similar a match
+            must be to be returned. The distance is measured between the
+            structural fingerprints in euclidean space.
+    """
+
+    def __init__(self,
+                 aflow_initial_ltol: float = 0.2,
```

Comment:
i would just call these variables "initial_ltol" instead of "aflow_initial_ltol", etc.

Reply:
Done